pandas/core/generic.py

```diff
@@ -2896,6 +2896,7 @@ def interpolate(self, method='linear', axis=0, limit=None, inplace=False,
             Update the NDFrame in place if possible.
         downcast : optional, 'infer' or None, defaults to None
             Downcast dtypes if possible.
+        kwargs : a dictionary of keyword arguments passed into interpolating function.
```

Comment:
This shouldn't be a dictionary I think? 
So just "Other keyword arguments are passed to the interpolating function"?